gravitee-apim-cypress/cypress/fixtures/fakers/applications.ts

```diff
@@ -19,9 +19,14 @@ import { Application } from '@model/applications';
 export class ApplicationFakers {
   static application(attributes?: Partial<Application>): Application {
     return <Application>{
-      ...attributes,
       name: faker.commerce.productName(),
-      description: faker.commerce.productDescription(),
+      description: 'This application was created by a Cypress test',
+      settings: {
+        app: {
+          // client_id: `client_${faker.random.alpha({ count: 5 })}`,
```

Comment:
Sounds a good idea to define a default random client_id here.
Can we uncomment it ? Or definitively remove it if not used ?

Reply:
Oh, right, I forgot to remove that one, I'll do that. I implemented that same faker call on a higher level, in the application test itself. Well, since it's optional to define a `client_id` and to keep the basic function as simple as possible I'd like to not keep it here but as mentioned defining it when needed.